AAI/AAIConnectProfile.md

```diff
@@ -4,7 +4,9 @@
 
 | Version | Date    | Editor                                     | Notes                   |
 |---------|---------|--------------------------------------------|-------------------------|
+| 1.0.1   | 2019-10 | David Bernick                              | Some clarifications     |
```

Comment:
Please make the "notes" more specific:

"Clarify that non-GA4GH claims are allowed in tokens"